public/app/features/dashboard/components/DashboardSettings/GeneralSettings.tsx

```diff
@@ -115,6 +117,12 @@ export function GeneralSettingsUnconnected({
             />
           </Field>
 
+          {config.featureToggles.nestedFolderPicker && (
```

Comment:
If we want to gate this Nested Folder picker under a separate feature toggle (I suppose to allow this to be disabled without disabling nested folders completely!), should we still make this field depend on both feature flags (i.e. nestedfolders too)? 
To avoid showing this picker when NestedFolder is off

Reply:
This is just an "internal" flag, so that we can work on multiple things is parallel, and still have it in main.

Reply:
@yaelleC @tskarhed the nested folders feature is useless without this select so I don't see a point in adding another feature toggle separate from nestedFolders

we have a FolderPicker component already I would replace this component when nestedFolders feature toggle is enabled.

Reply:
Torkel initial plan was they we would merge smaller increments of a non-functional folder picker and not break main.